File: 프로그래머스/unrated/181854. 배열의 길이에 따라 다른 연산하기/배열의 길이에 따라 다른 연산하기.cs
using System;

public class Solution {
    public int[] solution(int[] arr, int n) 
    {
        int i = arr.Length % 2 == 1 ? 0 : 1;
        for(; i < arr.Length; i += 2)
            arr[i] += n;
        return arr;
    }
}

File: 프로그래머스/0/181854. 배열의 길이에 따라 다른 연산하기/배열의 길이에 따라 다른 연산하기.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> arr, int n) {
    
    int i = arr.size() & 1 ? 0 : 1;
    for (; i < arr.size(); i+=2)
    {
        arr[i] += n;
    }
    
    return arr;
}